src/content/basics/api/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: API Access to Giant Swarm Resources
+description: An overview of the APIs that provide you with programmatic access to
+  resources like your tenant cluster in a Giant Swarm installation. Namely the Control
+  Plane Kubernetes API and the legacy Giant Swarm Rest API.
+date: 2020-05-04
+weight: 75
+type: page
+categories: ["basics"]
+last-review-date:
+---
+
+# API Access to Giant Swarm Resources
+
+For integrating Giant Swarm with your automation, we provide programmatic access via two different APIs:
+
+- [The Control Plane Kubernetes API](#cp-k8s-api)
```

Comment:
I'd turn this around.

The Control Plane API is in preview
The Giant Swarm Rest API is stable

Reply:
Ack